specification/hugr.md

```diff
@@ -834,6 +834,8 @@ statically-known number and type of elements, as does `Array<N>` (where
 N is a static constant). These types are also fixed-size if their
 components are.
 
+For integer types, the width is provided in the type, and signedness is left unspecified to be interpreted by operations. The width is allowed to be in the range [1,128].
```

Comment:
Were we going to restrict to powers of 2, i.e {2^0, 2^1, ..., 2^7} ?

Reply:
LLVM allows any value in the range [1, 2^23], but some passes require a power of 2 https://llvm.org/doxygen/classllvm_1_1IntegerType.html#a0a1d2bdb78754e9f271138e507f440e1a1ccefdf8a7414a6829f888e5071e0379